ert3/config/_experiment_run_config.py

```diff
@@ -0,0 +1,196 @@
+from typing import Dict, NamedTuple, cast
+
+import ert
+
+from . import (
+    EnsembleConfig,
+    ExperimentConfig,
+    ParametersConfig,
+    SourceNS,
+    StagesConfig,
+    Step,
+)
+
+
+class LinkedInput(NamedTuple):
+    """The LinkedInput class provides all necessary information to us an input
```

Comment:
Spelling mistake at the end of the line, I am at least not able to parse the meaning? :thinking: